fh-mbaas.js

```diff
@@ -121,11 +121,17 @@ function createAndSetLogger() {
 
   //Setting logger for fh-forms
   forms.init(logger);
-
-  //Setting global forms config
-  logger.debug("minsPerBackOffIndex", fhconfig.int('fhmbaas.dsMinsPerBackOffIndex'));
-  forms.core.setConfig({
-    minsPerBackOffIndex: fhconfig.int('fhmbaas.dsMinsPerBackOffIndex')
+  forms.setupSharedMongoConnections(logger, fhconfig.mongoConnectionString(), fhconfig.getConfig().rawConfig, function(err, sharedConnections) {
```

Comment:
I would be very hesitant to give the fh-forms module access to this administrative account.

I would advise setting up a user with access all database permissions but without "admin" permissions. E.g dropping databases etc.